### DISCUSSION TITLE
Cant log in CC? shadowban real? pepega?

### DISCUSSION THREAD

**Discussion by Neversong97** [chromiecraft/chromiecraft#7426] [General]
stuck on log in screen during "success" process I changed realmist, cleared cache, doesnt matter which account I try result is always the same. If anybody know anything know something that I dont then be a kind person and give info... thx :xdxd:

**Comment by heyitsbench** [comment]
Shadowbans like that aren't a thing, either you're banned or you're not. Can't say what your issue or solution specifically is though, I'm sorry to say.

**Reply by Neversong97** [reply]
Am pretty sure am not ingame, if there was possibility then id be banned on only my account not the ones Ive got access to... Any recommendation to solution?

**Comment by Nyeriah** [comment]
Is the realmlist set right?
Are you using the warmane client?

**Reply by Neversong97** [reply]
Well I been trying to log in like whole half of 2024. Yea its warmane client and am not 100% sure am like 1000% sure that my realmlist is fine swiching realmlist to other servers worked fine but I only encountered issue with CC. There is a post on github "cant connect to AC"

This is what describes my issue G https://github.com/azerothcore/azerothcore-wotlk/issues/5840 take a look but my issue is I cant connect to neither of servers. I can see the realms both live and PTR and when I try to connect process is stuck on success and stops there.

**Reply by Amrx2** [reply]
I can see you have logged on today so presuming that you fixed the issue?

**Reply by Neversong97** [reply]
I tried to log in today same thing happend, this been happening since after the summer of 2024 didnt pay much attention thought it was some minor stuff going on with the server. Nope issue still there

**Reply by Neversong97** [reply]
Key word is "I tried"

**Reply by Amrx2** [reply]
No idea sorry, checking your account I cannot even see you failing to connect to our authentication server so seemingly something is blocking you before then? Can you try via a VPN if it makes a difference?

**Reply by Helias** [reply]
did you try with a clean CC client?
